### PR TITLE
Enable basic model calibration

### PR DESCRIPTION
**Pull request by apulsipher**
Enables basic calibration of model based on accessed data

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 FROM rocker/tidyverse:4.4.1
 
 # Install epiworldR
-RUN install2.r epiworldR
+RUN installGithub.r UofUEpiBio/epiworldR
```
Comment (gvegayon) [inline .devcontainer/Dockerfile:6]: Your container is not re-building with the right version b/c it is caching the line. We can point to the specific commit we want to ensure it updates (and doesn't do caching):

```suggestion
RUN installGithub.r UofUEpiBio/epiworldR@cc673d1
```

File: index.qmd [hunk 2]
```diff
@@ -35,22 +36,225 @@ if (length(forecast_data) > 1) {
...
+  return(ifelse(sqrt(ans) < epsilon, 1.0, 0.0))
+}
+
+# Create the LFMCMC model
+lfmcmc_model <- LFMCMC(ef_model) |>
+  set_simulation_fun(lfmcmc_simulation_fun) |>
+  set_summary_fun(lfmcmc_summary_fun) |>
+  set_proposal_fun(lfmcmc_proposal_fun) |>
+  set_kernel_fun(lfmcmc_kernel_fun) |>
+  set_observed_data(forecast_data$Daily.Cases)
```
Comment (gvegayon) [inline index.qmd:223]: You should scale the data it get better performance. Unsure what is the proportion, but, if pop in Utah is 1,000,000 and we are running our model with 100,000, then we should do `forecast_data$Daily.Cases/10`:

```suggestion
# Create the LFMCMC model
obs_cases <- forecast_data$Daily.Cases/10

lfmcmc_model <- LFMCMC(ef_model) |>
  set_simulation_fun(lfmcmc_simulation_fun) |>
  set_summary_fun(lfmcmc_summary_fun) |>
  set_proposal_fun(lfmcmc_proposal_fun) |>
  set_kernel_fun(lfmcmc_kernel_fun) |>
  set_observed_data(obs_cases)
```

File: index.qmd [hunk 3]
```diff
@@ -35,22 +36,225 @@ if (length(forecast_data) > 1) {
...
+    size_of_peak,
+    sd_cases,
+    mean_cases
+  )))
+}
+
+lfmcmc_proposal_fun <- function(params_prev) {
+  res_1 <- abs(params_prev[1] + rnorm(1, mean = 0, sd = 0.5))
+  res_2_to_5 <- abs(params_prev[2:5] + rnorm(4, mean = 0, sd = 0.1))
+  res_6_to_7 <- abs(params_prev[6:7] + rnorm(2, mean = 0, sd = 1))
```
Comment (gvegayon) [inline index.qmd:194]: You should be using `plogis(qlogis() + rnorm())` approach I suggested for probabilities.

```suggestion
  res_1 <- abs(params_prev[1] + rnorm(1, mean = 0, sd = 0.5))
  res_2_to_5 <- abs(params_prev[2:5] + rnorm(4, mean = 0, sd = 0.1))
  res_6_to_7 <- abs(params_prev[6:7] + rnorm(2, mean = 0, sd = 1))
```

**Review by gvegayon — COMMENTED** [review]
Nice work! Is this the latest version? I have a handful of observations. LMK if you have any questions.

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (apulsipher) [reply .devcontainer/Dockerfile]: Thanks, this fixed the issue.

File: index.qmd (hunk 2 above)
Comment (apulsipher) [reply index.qmd]: Already done, the version you looked at here is out of date. Just committed the latest.

File: index.qmd (hunk 3 above)
Comment (apulsipher) [reply index.qmd]: Already done, the version you looked at here is out of date. Just committed the latest.

**Comment by apulsipher** [comment]
> Nice work! Is this the latest version? I have a handful of observations. LMK if you have any questions.

This wasn't the latest version, I've not committed the latest. Your suggestion for the Dockerfile fixed the issue I was having with epiworldR features. But the new changes broke something else in the code that I'm working on debugging now

**Comment by apulsipher** [comment]
@gvegayon I'm seeing unusually high recovery and transmission rates (rates > 1.0). Can you take a look and see what I'm missing?

**Pull request merged**